packages/odie-client/src/components/message/user-message.tsx

```diff
@@ -22,6 +22,7 @@ export const UserMessage = ( {
 
 	const hasCannedResponse = message.context?.flags?.canned_response;
 	const isRequestingHumanSupport = message.context?.flags?.forward_to_human_support;
+	const isOnlyMessage = message.context?.flags?.only_message;
```

Comment:
It took me a while to understand the idea behind this boolean.
I would suggest us to rename this variable. Maybe we should call it “hide-disclaimer” or something similar. WDYT?

Reply:
Plus, I think we could extract this into another place, this is getting too conditional and difficult to read/follow.

Reply:
One refactoring example:

```js
const renderExtraContactOptions = () => {
		return shouldUseHelpCenterExperience ? <GetSupport /> : extraContactOptions;
	};

	const renderDisclaimers = () => (
		<>
			<WasThisHelpfulButtons message={ message } isDisliked={ isDisliked } />
			<DirectEscalationLink messageId={ message.message_id } />
			<div className="disclaimer">
				{ __(
					"Generated by WordPress.com's Support AI. AI-generated responses may contain inaccurate information.",
					__i18n_text_domain__
				) }
				<ExternalLink href="https://automattic.com/ai-guidelines">
					{ __( 'Learn more.', __i18n_text_domain__ ) }
				</ExternalLink>
			</div>
		</>
	);

	return (
		<>
			<Markdown
				urlTransform={ uriTransformer }
				components={ {
					a: CustomALink,
				} }
			>
				{ isRequestingHumanSupport ? displayMessage : message.content }
			</Markdown>
			{ ! isOnlyMessage && (
				<>
					{ showExtraContactOptions ? renderExtraContactOptions() : isBot && renderDisclaimers() }
				</>
			) }
		</>
	);
```